README.md

```diff
@@ -13,6 +13,12 @@ This project is intended to allow groups of friends to organize game nights.
 
 For local development, [`just`](https://github.com/casey/just) is used to automate common workflows  (see [`justfile`](./justfile)).
 
+### Configuration
+
+#### SendGrid
+
+For email to work for login links, you will need a SendGrid token. You can find generate a new one under [Settings > API Keys](https://app.sendgrid.com/settings/api_keys) and store it in `app_config.toml` under the key `email_token`.
```

Comment:
Might docuent the option of the $EMAIL_TOKEN env var also to avoid tokens on disk.